controller/search_blackbox_test.go

```diff
@@ -1054,7 +1054,7 @@ func (s *searchControllerTestSuite) TestSearchQueryScenarioDriven() {
 		)
 		_, result := test.ShowSearchOK(t, nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
 		require.NotEmpty(t, result.Data)
-		assert.Len(t, result.Data, 1)
+		// assert.Len(t, result.Data, 1)
```

Comment:
What's the reason for commenting all these assertions out?

Reply:
Leftovers. Fixed.